swagger-spec/collection_submission_actions/list.yaml

```
@@ -0,0 +1,86 @@
+# /collection_submission_actions/
+post:
+  summary: Creates a Collection Submission Action
+  description: >-
+
+    This adds a Collection Submission Action to the Collection, causing the Collection state machine to preform a
+    transition.
+
+    #### Permissions
+
+    This information is only available to an admin on the Collection in question.
+
+    #### Returns
+
+    Returns a JSON object containing `data` and `links` keys.
+
+    The `data` key contains an array of nodes ids.
```

The POST doesn't return a list, right? Just the single collection_submission_action that was created?
```suggestion
    Returns a JSON object with a `data` key containing the representation of the requested collection submission action object, if the request is successful.
```